linodecli/baked/operation.py

```diff
@@ -463,10 +463,54 @@ def _add_args_post_put(self, parser) -> List[Tuple[str, str]]:
 
         return list_items
 
+    def _validate_parent_child_conflicts(self, parsed: argparse.Namespace):
```

Comment:
I originally wanted to implement this validation using argparse's mutually exclusive groups but I couldn't get it to play nicely with multiple compatible fields.